### PR TITLE
feat: Rename some API parameters

### LINKED ISSUE #126: goto: Rename `disableJavaScript` into `javascript`

### LINKED ISSUE #127: goto: Rename `disableAnimations` into `animations`

### LINKED ISSUE #129: goto: Rename `media` into `mediaType`

### PR DESCRIPTION
**Pull request by adityawankhede5**
Closes #126
Closes #127
Closes #129

### CODE REVIEW

**Comment by Kikobeats** [comment]
What do you think into rename `javascript` to `scripts`?

Then, passing `scripts=false` should disable JavaScript on the page.

File: packages/goto/src/index.js [hunk 1]
```diff
@@ -56,13 +56,13 @@ module.exports = ({ timeout, ...deviceOpts }) => {
 
   const goto = async (
     page,
-    { url, media, adblock = true, headers = {}, waitFor = 0, disableAnimations = true, disableJavaScript = false, ...args }
+    { url, mediaType, adblock = true, headers = {}, waitFor = 0, animations = true, javascript = false, ...args }
```
Comment (remusao) [inline packages/goto/src/index.js]: It seems a bit confusing to me that passing “JavaScript=true” will actually disable JavaScript in the page.

Comment (adityawankhede5) [reply packages/goto/src/index.js]: The script only needs to disabled if it is running, and if script is running the status of the **javascript** variable must be true.
So it makes perfect sense to me that setting it true will eventually disable it.

Comment (Kikobeats) [reply packages/goto/src/index.js]: resolved at https://github.com/microlinkhq/browserless/pull/130/commits/9ff0a3f3ade9942e1e5dd353c25d235ab44f5001

**Comment by Kikobeats** [comment]
@adityawankhede5 Thanks for this, released at [v5.20.0](https://github.com/microlinkhq/browserless/releases/tag/v5.20.0)

Welcome as a browserless contributor 🎉

**Pull request merged**